recipes/Libri-Light/self-supervised-learning/librilight_prepare.py

```diff
@@ -0,0 +1,270 @@
+"""
+Data preparation.
+
+1. Download the Libri-Light dataset through the toolkit in the Libri-Light github repo
+    "https://github.com/facebookresearch/libri-light/"
+
+2. Use the data_preparation/cut_by_vad.py script of the Libri-Light repo to do the vad. For example,
+    "python cut_by_vad.py --input_dir path_to_Libri-Light/small --output_dir Libri-Light_vad/small_vad --target_len_sec 20"
+
+Author
+------
+ * Mirco Ravanelli, 2020
+ * Ju-Chieh Chou, 2020
+ * Loren Lugosch, 2020
+ * Pierre Champion, 2023
+ * Adel Moumen, 2024
```

Comment:
Not sure about hwo relevants those people are with respect to this file. If they didnt contributed to this file, then you can remove them